projects/core/src/user/occ/user.service.spec.ts

```diff
@@ -256,4 +257,28 @@ describe('OccUserService', () => {
       mockReq.flush('');
     });
   });
+
+  describe('update password: ', () => {
+    it('should update the password for userId', () => {
+      const userId = 'test@test.com';
+      const oldPassword = 'OldPass123!';
+      const newPassword = 'NewPass456!';
+
+      service
+        .updatePassword(userId, oldPassword, newPassword)
+        .subscribe(result => expect(result).toEqual(''));
```

Comment:
Maybe move the `expect` out of the `subscribe` block?

Reply:
Agreed, the `expect` should not be in the `subscribe` block.

Fixed.